prac_06/car.py

```diff
@@ -0,0 +1,35 @@
+"""CP1404/CP5632 Practical - Car class example."""
```

Comment:
Good work. It would be good if some of the docstrings are a bit longer to explain the overall program or class.